__tests__/setup-go.test.ts

```diff
@@ -230,6 +231,22 @@ describe('setup-go', () => {
     expect(logSpy).toHaveBeenCalledWith(`Setup go version spec 1.13.0`);
   });
 
+  it('does not export any varibles', async () => {
```

Comment:
```suggestion
  it('does not export any variables', async () => {
```

Reply:
FYI you can just commit suggestions directly from the GitHub UI:

<img width="481" alt="Screen Shot 2022-03-11 at 2 10 50 PM" src="https://user-images.githubusercontent.com/33549821/157935596-7e87cd0e-915a-474b-b866-a16ff6385e43.png">

Reply:
Fixed with the changes in https://github.com/actions/setup-go/pull/175/commits/2a34c33bd7648b12fa2caabb0bfc8919e37aacc6